account_avatax/models/avatax_rest_api.py

```diff
@@ -305,7 +305,7 @@ def get_tax(
         # Enrich Avatax result with Odoo tax computation
         for line in result.get("lines", []):
             line["rate"] = (
-                round(sum(x["rate"] for x in line["details"]) * 100, 4)
+                round(sum(x["rate"] for x in line["details"] if "tax" in x and x["tax"] > 0) * 100, 4)
```

Comment:
Making it more Pythonic.
```suggestion
                round(sum(x["rate"] for x in line["details"] if x.get("tax")) * 100, 4)
```

I wish we had a test case for this line of code.
Isolating this bit of code in a helper function could make that easier to do.